specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/subscriptions.json

```diff
@@ -0,0 +1,730 @@
+            "Other"
+          ],
+          "x-ms-enum": {
+            "name": "RegionCategory",
+            "modelAsString": true
+          }
+        },
+        "geographyGroup": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The geography group of the location."
+        },
+        "longitude": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The longitude of the location."
+        },
+        "latitude": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The latitude of the location."
+        },
+        "physicalLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The physical location of the Azure location."
+        },
+        "pairedRegion": {
+          "type": "array",
+          "description": "The regions paired to this region.",
+          "items": {
+            "$ref": "#/definitions/PairedRegion"
+          }
+        },
+        "homeLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The home location of an edge zone."
+        }
+      }
+    },
+    "Location": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified ID of the location. For example, /subscriptions/00000000-0000-0000-0000-000000000000/locations/westus."
+        },
+        "subscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription ID."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The location name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The location type.",
+          "enum": [
+            "Region",
+            "EdgeZone"
+          ],
+          "x-ms-enum": {
+            "name": "LocationType",
+            "modelAsString": false
+          }
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the location."
+        },
+        "regionalDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the location and its region."
+        },
+        "metadata": {
+          "description": "Metadata of the location, such as lat/long, paired region, and others.",
+          "$ref": "#/definitions/LocationMetadata"
+        }
+      },
+      "description": "Location information."
+    },
+    "LocationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Location"
+          },
+          "description": "An array of locations."
+        }
+      },
+      "description": "Location list operation response."
```

Comment:
next link?

Reply:
unfortunately we don't support paging on this API yet.